File: BubbleSort/BubbleSort-Csharp/src/Sort.cs
using System;
using System.Collections.Generic;

namespace Sort 
{
  class Sort 
  {
    public void bubble_sort(List<int>arr)
    {
      int n=arr.Count+1;

      for(int i=1;i<n;i++)
      {
        for(int j=0;j<n-i-1;j++)
        {
          if(arr[j]>arr[j+1])
          {
            int temp=arr[j];
            arr[j]=arr[j+1];
            arr[j+1]=temp;
          }
        }
      }

      foreach(int i in arr)
      {
        Console.Write(i);
      }
    }
  }
}

File: SelectionSort/SelectionSort-py/src/Sort.py

def  sort(arr:list[int])->None:
    n:int=len(arr)

    for i in range(0,n,1):
        for j in range(i+1,n,1):
            min:int=i
            if arr[j]<arr[min]:
                min=j

            temp:int=arr[i]
            arr[i]=arr[min]
            arr[min]=temp
